src/sonic-yang-models/yang-templates/sonic-acl.yang.j2

```diff
@@ -117,7 +116,6 @@ module sonic-acl {
 					}
 
 					case ip6_prefix {
-						when "boolean(IP_TYPE[.='ANY' or .='IP' or .='IPV6' or .='IPv6ANY'])";
```

Comment:
without this condition, user can apply ip6_prefix with IP_TYPE as IPv4.

So is it possible to modify the condition such that if IP_TYPE does not exist, this results in True.
If exists then it check the value as today. 

Let me know if that is not possible then I will approve this PR with current Changes itself. Thanks

Reply:
It's a good idea. I will investigate the reply shortly.

Reply:
@praveen-li  I updated the yang model. Now if IP_TYPE does not exist, this results in True. Please help review. Thanks